packages/manifest/src/generated-types/index.ts

```diff
@@ -97,7 +97,7 @@ export type TeamsManifest =
 
 export type TeamsManifestLatest = TeamsManifestV1D21.TeamsManifestV1D21;
 
-export { SensitivityLabel } from "./copilot/declarative-agent/DeclarativeAgentManifestV1D4";
+export { SensitivityLabel } from "./copilot/declarative-agent/DeclarativeAgentManifestV1D6";
 
 export type DeclarativeAgentManifest =
```

Comment:
Is this variable used? It's the union of v1.0-v1.6. Same for DeclarativeAgentManifestLatest